bin/runtime/src/lib.rs

```diff
@@ -532,10 +554,10 @@ impl pallet_utility::Config for Runtime {
 
 parameter_types! {
     pub const DepositPerItem: Balance = deposit(1, 0);
-    pub const DepositPerByte: Balance = deposit(0, 1);
+    pub const DepositPerByte: Balance = 1_000_000_000; // 1e-3 alepha (1 milli aleph)
```

Comment:
why? `deposit(0,1)` is exactly the price for a byte storage which we use e.g. for multisig. I agree, that probably it is not good having such universal, single cost calculation method, but we use it also one line above for `DepositPerItem`, so it is kinda confusing :|

also, Polish comment :stuck_out_tongue_closed_eyes:

Reply:
well, there's a discussion to be had here: 

- with the default settings we had something like 1e11 / byte
- flipper (release optimized) is ~1.8KB (unoptimized is like 50KB), 1.8e14 pico-salpeh's total
- a shot from faucet is 9001 saleph's, so it's what? ~50 flippers.
- Since we want to promote experimentation this is raising it 100x 

Flipper is an ultra basic contract, I have no idea how WASM size increases with code complexity, but it won't be linear I'd imagine...